modules/localgov_expired_events/src/Form/ExpiredEventSettingsForm.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * @file
+ * Contains Drupal\localgov_expired_events\Form\ExpiredEventSettingsForm.
+ */
+
+namespace Drupal\localgov_expired_events\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+class ExpiredEventSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'localgov_expired_events_form';
+  }
+
+  /**
+   * {@inheritdoc }
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildForm($form, $form_state);
+
+    $config = $this->config('localgov_expired_events.settings');
```

Comment:
You can simplify this by using the new #config_target property. See https://www.qed42.com/insights/exploring-the-new-config-target-option-in-drupal-10